src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.csproj

```diff
@@ -4,22 +4,26 @@
 
   <PropertyGroup>
     <Description>Build-time references required to enable Razor view compilation as part of building the application.</Description>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <RuntimeIdentifier Condition="!$(TargetFramework.StartsWith('netcoreapp'))">win7-x64</RuntimeIdentifier>
     <PackageTags>cshtml;razor;compilation;precompilation;aspnetcore</PackageTags>
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <OutputType>exe</OutputType>
-    <!-- Include the build outputs in the build directory (and not the lib directory) -->
-    <BuildOutputTargetFolder>build</BuildOutputTargetFolder>
+    <X86ProjectDirectory>..\..\tools\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation-x86\</X86ProjectDirectory>
   </PropertyGroup>
 
   <ItemGroup>
     <None Include="build\**\*" Pack="true" PackagePath="%(Identity)" />
+    <None Include="buildMultiTargeting\*" Pack="true" PackagePath="%(Identity)" />
+    <None Include="$(X86ProjectDirectory)\bin\$(Configuration)\net461\win7-x86\$(MSBuildThisFileName)-x86.exe" Pack="true" PackagePath="lib\net461\$(MSBuildThisFileName)-x86.exe" />
```

Comment:
@pranavkm The scaffolding guys handle x86 by having their own tool have a x86 RID. Since it's compatible with x64 all just works. Is there a reason why we did this approach for so long?

Leaving as is for now because it's worked in the past; however, these bits would be significantly simpler without the extra project/x86ness.